src/quo2/components/drawers/drawer_buttons/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.drawers.drawer-buttons.view
+  (:require [react-native.core :as rn]
+            [quo2.components.icon :as icon]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.drawers.drawer-buttons.style :as style]))
+
+(defn render-bottom
+  [children]
+  [rn/view
+   {:flex-direction  :row
+    :justify-content :space-between}
+   [children]
+   [rn/view
+    {:style {:border-radius   40
+             :border-width    1
+             :margin-left     24
+             :height          28
+             :width           28
+             :justify-content :center
+             :align-items     :center
+             :border-color    (colors/alpha colors/white 0.05)}}
+    [icon/icon :arrow-right
+     {:color colors/white
+
+      :size  20}]]])
+
+(defn render-children-bottom
+  [children]
+  (if (or (string? children) (keyword children))
+    [render-bottom
+     (fn []
+       [text/text
+        {:size   :paragraph-2
+         :style  {:flex  1
+                  :color (colors/alpha colors/white 0.7)}
+         :weight :semi-bold}
+        children])]
+    [render-bottom children]))
+
+(defn render-children-top
+  [children]
```

Comment:
I'd say it's more comfortable to pass children as a hiccup in ClojureScript instead of passing a render-fn. If children were passed as hiccup, you won't need to wrap children in `[ ]`, and it'd be more natural to pass children to this component.

Reply:
yep, will do! :)